.github/workflows/ci.yml

```diff
@@ -44,7 +51,7 @@ jobs:
 
     - name: Configure Cabal
       run: |
-        cp .github/workflows/cabal.project.local .
+        cp .github/workflows/cabal.project.${{ matrix.variant }}.local ./cabal.project.local
```

Comment:
If you move the project.local file, then the import won't work right? The import like should say `import: .github/workflows/cabal.project.default.local`, no?

Reply:
yup, you're right, fixed it